content/en/monitors/guide/alert_aggregation.md

```diff
@@ -0,0 +1,89 @@
+  tag: "Documentation"
+  text: "Learn more about configuring notifications and automations"
+- link: "/monitors/guide/monitor_aggregators/"
+  tag: "Documentation"
+  text: "Monitor Aggregators"
+---
+
+## Overview
+
+Ensuring the health and performance of systems requires timely and actionable notifications. However, the effectiveness of alerts isn't just about when they are triggered but also about how many alerts are created. This is where alert aggregation becomes critical. 
+
+Let's say you're responsible for an e-commerce website. When an issue arises, do you want a single alert summarizing the problem or multiple but specific alerts pin-pointing every aspect of the failure?   
+The answer will depend on your system's architecture, the nature of the issue, and your team's workflow but knowing how to aggregate the alerts will help ensure teams are alerted to issues while not receiving more notification than necessary. 
+
+This guide explores the different alert aggregation capabilities and patterns to use in a variety of circumstances.
+
+## Types of Alert Aggregation 
+
+By default, Datadog will send an alert per monitored group. You can however change that to a single notification sent no matter how many monitored groups breaches the threshold.
+
+Let's take this example:
+
+{{< img src="/monitors/guide/alert_aggregation/monitor_query_multi_alert.png" alt="Example monitor query grouped by topic and partition" style="width:100%;" >}}
+
+### Simple Alert
+
+A Simple Alert monitor is a monitor that will aggregate your alerts into a single unique alert.
+{{< img src="monitors/guide/alert_aggregation/simple_alert_notification.png" alt="Notification configuration for a simple alert monitor" style="width:100%;" >}}
+
+Using the example, no matter which topic or partition breaches the threshold the monitor will alert. All notifications are aggregated into one single alert.
+
+For more information, see [Configure Monitors - Simple alert][1].
+
+### Multi Alert
+
+A Multi Alert monitor is a monitor that triggers for each unique combination of groups.  
+{{< img src="/monitors/guide/alert_aggregation/multi_alert_notification.png" alt="Notification configuration for a multi alert monitor" style="width:100%;" >}}
+In our case, the monitor will send a notification each time a combination of topic and partition breaches the threshold.  
+When using Multi Alert, using [Variables][2] become very useful to get more granularity in your notifications. They help you get a specific message depending on the group that triggers the alert.
+
+For more information, see [Configure Monitors - Multi alert][3].
+
+## Use cases
+
+{{% collapse-content title="Simple Alert" level="h4" expanded=false %}}
+#### Scenario
+You're monitoring a Kafka-based logging system with "error-logs" and "user-events". If any partition gets a message lag of more than 500, you want to know about it, but you don't need multiple alerts if multiple partitions are lagging.  
```

Comment:
Are error-logs and user-events the group by attributes? What does partition mean here?

```suggestion
You're monitoring a Kafka-based logging system grouped by `error-logs` and `user-events`. If any partition gets a message lag of more than 500, you want to know about it, but you don't need multiple alerts if multiple partitions are lagging.  
```